packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -53,7 +53,7 @@ export default util.createRule<Options, MessageIds>({
     const checker = parserServices.program.getTypeChecker();
 
     return {
-      MemberExpression(node) {
+      [AST_NODE_TYPES.MemberExpression](node: TSESTree.MemberExpression) {
```

Comment:
If the type isn’t inferred anymore, IMO the original way is better (cc @mysticatea)

Reply:
It's a bug of the type definition, I think.

The `MemberExpression` is a string to compare with `Node#type`, so it should be an enum value if you decide to use enums.

Of course, I'd like to recommend that don't use enums for node types at all.

Reply:
Went back to just using the literal. ✔